packages/jokul/src/components/text-input/styles/text-input.scss

```diff
@@ -187,6 +182,12 @@ $_text-input-selection-color--inverted: color.scale(
         }
     }
 
+    &:focus-visible {
+        // The default icon-button focus looks pretty terrible when the button
+        // is inside an input field so make sure to remove it.
+        outline: none !important;
+    }
+
```

Comment:
Jeg lurer på om vi heller bør endre stilene for text-input litt for å ta høyde for at vi nå sender inn en `IconButton`. Ser også ut som vi bør sette `padding: 0` på `IconButton`, siden den nå får "standard" knappe-padding fra nettleseren.

Hvis vi fjerner paddingen i `IconButton` og heller legger til padding i `TextInput` hvis den har en action går vi fra dette
![Skjermbilde 2024-12-09 kl  12 36 59](https://github.com/user-attachments/assets/67bacd48-46ef-4562-a770-1b623b956b96)
til dette
![Skjermbilde 2024-12-09 kl  12 38 37](https://github.com/user-attachments/assets/3bc13973-1ed1-450c-bc92-2686dbabb64b)

Reply:
Det ser smoothere ut ja, kanskje det løser #4347 i samme slengen. Tester.

Reply:
Eller det vil si, jeg henger ikke helt med her. Ditt "etter" bilde er sånn det ser ut hos meg i denne PRen uten noen endringer. Du må ha gjort noe mer enn å bare flytte paddingen også?

Reply:
@piofinn jeg har litt lyst til å la dette ligge til vi uansett går over focus-håndteringen i alle komponenter. Det ser allerede bedre ut nå enn det gjorde.